geocoder_tester/world/france/iledefrance/test_addresses.csv

```diff
@@ -512,9 +512,6 @@ query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_
 12BIS Chemin d'Orveau Vayres-sur-Essonne,,,,12BIS Chemin d'Orveau,,,,91820
 12BIS Chemin d'Orveau 91820,,,,12BIS Chemin d'Orveau,,,Vayres-sur-Essonne,
 12BIS Chemin d'Orveau,2.351197,48.438203,,12BIS Chemin d'Orveau,,,,91820
-3 Avenue Lyautey Vert-le-Petit,,,,3 Avenue Lyautey,,,,91710

```

Comment:
It's rather unclear if the expected name of this street is "Avenue Lyautey", "Avenue du Maréchal Lyautey" or even "Avenue du Matelot Lyautey" ...